arctic/store/_pickle_store.py

```diff
@@ -54,12 +56,16 @@ def write(self, arctic_lib, version, symbol, item, previous_version):
         collection = arctic_lib.get_top_level_collection()
         # Try to pickle it. This is best effort
         version['blob'] = _MAGIC_CHUNKED
-        pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        try:
+            pickled = lz4.compressHC(cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL))
+        except OverflowError:
+            pickled = cPickle.dumps(item, protocol=cPickle.HIGHEST_PROTOCOL)
+            version['blob'] = _MAGIC_CHUNKED_NO_COMPRESS
 
         for i in xrange(int(len(pickled) / _CHUNK_SIZE + 1)):
             segment = {'data': Binary(pickled[i * _CHUNK_SIZE : (i + 1) * _CHUNK_SIZE])}
-            sha = checksum(symbol, segment)
             segment['segment'] = i
+            sha = checksum(symbol, segment)
```

Comment:
i dont think there is any question that data chunks are identical -- which is the issue, it needs to write X of those, and each time they overwrite eachother!